pybloom_live/pybloom.py

```diff
@@ -291,6 +291,43 @@ def add(self, key):
         filter.add(key, skip_check=True)
         return False
 
+    def copy(self):
+        """Return a copy of this scalable bloom filter.
+        """
+        new_filter = ScalableBloomFilter(initial_capacity=self.initial_capacity,
+                                         error_rate=self.error_rate,
+                                         mode=self.SMALL_SET_GROWTH)
```

Comment:
I agree that using `deepcopy`  will look neater. However, we also need to change it for `BloomFilter` class to keep it consistent?